src/main/java/dev/dfonline/codeclient/config/Config.java

```diff
@@ -372,6 +374,18 @@ public YetAnotherConfigLib getLibConfig() {
                                 )
                                 .controller(TickBoxControllerBuilder::create)
                                 .build())
+                        .option(Option.createBuilder(boolean.class)
+                                .name(Text.literal("Build Phase Toggle"))
+                                .description(OptionDescription.createBuilder()
+                                        .text(Text.literal("If Build Phaser should be a toggle instead of a hold."))
```

Comment:
New config options should use `Text.translatable` for text.
You can put the translation in `en_us.json`, put it near other config translations